### PR TITLE
issue #104, issue #101, issue #106  #97 and #58 resolved

### PR DESCRIPTION
**Pull request by shubhamkakkar**
The new codes for the respective components could be found under the folder *src* of folder *shubhamkakkar* 

To see the working of the components on Expo Application, refer https://github.com/shubhamkakkar/galio/tree/examples

### CODE REVIEW

**Comment by palingheorghe** [comment]
@shubhamkakkar Please create another PR specifically for #58 as seen in our [Working on a PR guide](https://github.com/galio-org/galio/wiki/Working-on-a-PR).

**Comment by shubhamkakkar** [comment]
hey, I am sorry for the inconveniences caused, I will soon raise a new PR
for issue #58 as well. This particular PR is having some conflicts I
suppose and doesn't follow the PR pattern of *galio*

for most of the other issues mentioned to be resolved by this PR, i have
raised custom PR for each of them so that you can check each working
individually!!

Issue #58 's PR will be raised in a day or two ( PS: actually my exams are
going on 😅 )

**Comment by palingheorghe** [comment]
Hi @shubhamkakkar ! When are we having the PR for issue #58 ? You pushed everything else besides that one. 

thanks! ❤️

**Comment by palingheorghe** [comment]
Hi @shubhamkakkar ! We still didn't got all the PRs.. The PRs for issue #101 , #106, #97 and #58 are still missing. 

As a priority I'd prefer if you'd follow the next priority list:
1. #58 is the most important one for v0.6, you've pushed it here and then sent me a Snack link but I can't verify it so I'd actually prefer a PR. (I'm going to keep trying with that Snack link but honestly, a work-in-progress PR is a lot better)
2. #101 
3. #97 
4. #106 

Honestly no. 2, 3, 4 are a lot less important than the no.1 item on the priority list.

Thank for your help ❤️🔥 and please let's focus on creating more organized PRs because this has gotten quite out of hand for me, it's starting to be really disorganized. 

tl;dr: #58 is the most important PR right now.

**Comment by shubhamkakkar** [comment]
@palingheorghe  #125 is a new PR dealing only with Segment !! its a work in progress
